kmir/src/kmir/kdist/mir-semantics/rt/data.md

```diff
@@ -1067,6 +1067,30 @@ Type casts between a number of different types exist in MIR.
 
 ```k
   syntax Evaluation ::= #cast( Evaluation, CastKind, MaybeTy, Ty ) [strict(1)]
+  syntax Evaluation ::= castAux ( Value, CastKind, MaybeTypeInfo, TypeInfo ) [function]
```

Comment:
Maybe add `no-evaluators` to this function because it does not have equations, only simplifications